tests/server/server-executeReadWriteCommand-001.phpt

```diff
@@ -11,7 +11,7 @@ require_once __DIR__ . "/../utils/basic.inc";
 require_once __DIR__ . "/../utils/observer.php";
 
 $manager = new MongoDB\Driver\Manager(URI);
-$server = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY));
+$server = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY));
```

Comment:
There must be a reason why RP_SECONDARY worked before — do you know why that is?

Reply:
If the topology type is single (i.e. standalone), [`mongoc_topology_description_select()` ignores the read preference](https://github.com/mongodb/mongo-c-driver/blob/865e11a3498240dbd927e537dc3f212049f901fb/src/mongoc/mongoc-topology-description.c#L800). This conforms to the server selection spec's rules for [Topology type: Single](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#topology-type-single).

Reply:
Ah! :+1: